bytecode/src/function/instructions/mod.rs

```diff
@@ -92,6 +101,32 @@ pub enum Instruction<P: Program> {
     Sub(Sub<P>),
     /// Subtracts `second` from `first`, wrapping around at the boundary of the type, and storing the outcome in `destination`.
     SubWrapped(SubWrapped<P>),
+    /// Performs a Pedersen hash taking a 64-bit value as input.
```

Comment:
nit: these need to be ordered alphabetically, otherwise, our opcode IDs will fail ordering tests

Reply:
nit: rename to `HashPed*`, `HashPsd*`, and `CommitPed*` so that they are indexed correctly by their opcode mnemonic